patterns/2-structured/fixing-issues-step-by-step.md

```diff
@@ -0,0 +1,43 @@
+## Title
+
+Fixing Issues Step by Step
+
+## Patlet
+
+Solving an issue by deadline without consideration of the current scope of change to process causes developer frustration and decreased efficiencies. Utilize a continuous improvement approach which right-sizes the work and provides realistic expectations.
+
+## Problem
+
+Results are expected to take place within a few days/weeks after a process has started. Adding a process or a toolchain by itself will not provide any result by itself beyond developer frustration. Stating that something has to work by a certain deadline will not make it possible, this will just force the situation and workarounds will be done by developers.
+
```

Comment:
```suggestion

## Context

* Developer frustration is spread and at high levels. Complaints about pressure and stress.
* Clumpsy workarounds, undocumented and/or prone to be broken/voided are frequent.
* Voluntary contributions are scant.
```

Reply:
As there is no Context section yet, let's just add this in and improve it incrementally.